src/themes/oui-cascadia/oui_cascadia_colors_dark.scss

```diff
@@ -9,24 +9,41 @@
  * GitHub history for details.
  */
 
-@import '../oui/oui_colors_dark.scss';
+@import 'global_styling/functions/index';
```

Comment:
Why not just point to default theme global styling? Will we need to update these (mixing, functions, etc) when implementing the theme?

Reply:
Donno. We might. By having a separate one, we can safely make any change needed without having to worry about it. Since these files are not the ones going to the users' browsers, having duplicates while we iterate is not "wrong".